modules/atomic-widgets/base/style-transformer-base.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Elementor\Modules\AtomicWidgets\Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+abstract class Style_Transformer_Base {
```

Comment:
I think this will be handle by my side, but we can talk about it

Reply:
Totally agree, it's a placeholder for now but I based it on your work
One thing I need to align with you on is how can I invoke the value-transform logic from within a transformer (I passed a callback as its second argument)

Reply:
Created [this task](https://elementor.atlassian.net/browse/EDS-424) to refactor once the main backend transformers task is done